packages/nx-openapi/README.md

```diff
@@ -0,0 +1,74 @@
+# openapi-plugin
+
+Generate typed REST clients from OpenAPI 3.0.0 Specifications.
```

Comment:
OpenAPI spec is up to 3.1.1 now - do we support 3.0.0 exactly, 3.X, or something else?